sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3532,5 +3533,4 @@ _ZNK2cl4sycl9exception11get_contextEv
 _ZNK2cl4sycl9exception11has_contextEv
 _ZNK2cl4sycl9exception4whatEv
 __sycl_register_lib
-__sycl_unregister_lib
-
+__sycl_unregister_lib
```

Comment:
Minor comment: is this last line removal needed here? It does not seem to be liked by github. Please add/return the new-line-at-the-end-of-file.

Reply:
Yes, I will do that.

Reply:
Done.